.github/workflows/build-ubuntu20.04-AZURE.yml

```diff
@@ -1,27 +1,13 @@
 name: build-ubuntu-20.04-AZURE
 on:
-  push:
-    branches:
-      - dev
-      - release-*
-      - refs/tags/*
-    paths-ignore:
-      - '.github/workflows/quarto-render.yml'
-      - '_quarto.yml'
-      - 'quarto-materials/*'
-      - '**/.md'
-      - 'doc/source/conf.py'
-      - 'tiledb/sm/c_api/tiledb_version.h'
-  pull_request:
-    branches:
-      - '*'  # must quote since "*" is a YAML reserved character; we want a string
-    paths-ignore:
-      - '.github/workflows/quarto-render.yml'
-      - '_quarto.yml'
-      - 'quarto-materials/*'
-      - '**/.md'
-      - 'doc/source/conf.py'
-      - 'tiledb/sm/c_api/tiledb_version.h'
+  workflow_call:
+    inputs:
+      job_name:
+        description: 'Name of the job for display purposes'
```

Comment:
Do we want a name?

Reply:
Deleting this file, no longer necessary.